### ISSUE TITLE
Nan for softmax(pre_logits)

### ISSUE THREAD

**Issue by wjduand** [nyu-cv-fa19/SSLGANjr#12]
Got nan for rotation probabilities when training for a while.
